app/models/supplejack_api/search.rb

```diff
@@ -93,6 +93,8 @@ def search_builder
         search = QueryBuilder::WithBoudingBox.new(search, options.geo_bbox).call
         search = QueryBuilder::SolrQuery.new(search, options.solr_query).call
         search = QueryBuilder::FacetPivot.new(search, options.facet_pivots).call
+        search = QueryBuilder::FacetRange.new(search, options.facet_range, options.facet_range_start,
```

Comment:
question: wondering if there was any thought of referencing a singular query builder which would in turn call all of these?

Reply:
I'm not sure what the original intention of this class was but I do agree it is a bit strange to look at! This is a very old part of Supplejack